pom.xml

```diff
@@ -53,109 +53,134 @@
         <artifactId>snet-sdk-plugin-core</artifactId>
         <version>${project.version}</version>
       </dependency>
+
       <!-- protobuf + grpc begin -->
+      <!-- https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java -->
       <dependency>
         <groupId>com.google.protobuf</groupId>
         <artifactId>protobuf-java</artifactId>
-        <version>${protobuf.version}</version>
+        <version>4.31.1</version>
```

Comment:
This version is shared across few locations Please update `properties` above instead of updating specific `version` tags.